types/block.go

```diff
@@ -34,13 +34,34 @@ const (
 	MaxAminoOverheadForBlock int64 = 11
 )
 
+// DataAvailabilityHeader contains the row and column roots of the erasure
+// coded version of the data in Block.Data.
+// Therefor the original Block.Data is arranged in a
+// k × k matrix, which is then "extended" to a
+// 2k × 2k matrix applying multiple times Reed-Solomon encoding.
+// For details see Section 5.2: https://arxiv.org/abs/1809.09044
+type DataAvailabilityHeader struct {
+	// RowRoot_j 	= root((M_{j,1} || M_{j,2} || ... || M_{j,2k} ))
+	RowsRoots [][]byte
+	// ColumnRoot_j = root((M_{1,j} || M_{2,j} || ... || M_{2k,j} ))
+	ColumnRoots [][]byte
```

Comment:
If there's not a type for hash digests then we should add one. Is there is then `[]byte` can be replaced with it.

Reply:
There is none. At least a type alias would be good! Will add in a followup PR!